languages/tlaplus-grammar.json

```diff
@@ -170,6 +170,9 @@
                 }
             },
             "patterns": [
+                {
```

Comment:
I know little about VSCode.  Is this what fixes the syntax highlighting?

Reply:
Yes. As you mentioned in the issue, a regex is being used to identify a module but there are actually two regexes, one for the start token and one for the end (see line 151 and 166). This section define the patterns that can be found inside the body, by adding #module I'm allowing recursively searching for modules inside the module.

Reply:
Could you add that explanation to the git commit message?